include/utils/PhysicalConstants.h

```diff
@@ -0,0 +1,29 @@
+/************************************************************/
+/*                DO NOT MODIFY THIS HEADER                 */
+/*   TMAP8: Tritium Migration Analysis Program, Version 8   */
+/*                                                          */
+/*   Copyright 2021 - 2023 Battelle Energy Alliance, LLC    */
+/*                   ALL RIGHTS RESERVED                    */
+/************************************************************/
+
+#pragma once
+
+namespace PhysicalConstants
+{
+// Physical constants
+// Avogadro's number (at/mol)
```

Comment:
```suggestion
// Avogadro's number (atoms/mol)
```
Prefer to type `at` out for this one